src/InsertCapableInterface.php

```diff
@@ -17,9 +19,9 @@
      *
      * @since [*next-version*]
      *
-     * @param array $records An array of record data sets, each mapping field name to value.
+     * @param array[]|ArrayAccess|stdClass|ContainerInterface|Traversable $records A list of record data containers.
```

Comment:
Perhaps widen the type of elements of the array to reflect that it can be an array of any containers, and mention that in the doc.

Reply:
You mean, to reflect that the container types in the array can be mixed?

Reply:
Not sure how you could reflect that. Right now, it accepts an array of arrays. What more does it accept an array of?

Reply:
Well, an array/traversable of arrays, `stdClass` instances, `ArrayAccess` instances or `ContainerInterface` instances.

Reply:
Graet! But the documented type does not reflect that.

Reply:
Suggestions then?

Reply:
See 973c6e2552bbc12003a09b53d727355e142ce932.